tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -174,10 +175,12 @@ $env:ASPNETCORE_URLS="http://*3331;https://*:8881"  // set custom ports for both
 
 #### Input arguments
 
-Use command line argument `--urls` to bind to a non-default host and port. This configuration will override the environment configuration.  For example, to only bind to localhost http 5000, provide the argument `--urls http://localhost:9000`.
+When starting the test proxy there are Verbs, as shown in the test-proxy --help output above. For the start  verb, there can be additional command arguments that need to get passed to **Host.CreateDefaultBuilder** 'as is'. The way this is done is through the use of `--` or dashdash as it's referred to. If `--` is on the command line, everything after will be treated as arguments that will be passed into this Host call.
+
+For example, you can use command line argument `--urls` to bind to a non-default host and port. This configuration will override the environment configuration. To bind to localhost http 5000, provide the argument `-- --urls http://localhost:9000`. Note that `--`, space, then the `--urls http://localhost:9000`.
 
 ```powershell
-test-proxy --urls "http://localhost:9000;https://localhost:9001"
+test-proxy -- --urls "http://localhost:9000;https://localhost:9001"
```

Comment:
To me this feels confusing/un-intuitive. Is it only `urls` that needs to get passed to the inner host builder? If we control the outer cli wrapper, why is this necessary?

Reply:
@benbp, there's a whole bunch of random items that can be passed to Host.CreateDefaultBuilder. Normally, this is done by just passing args directly to the Host.CreateDefaultBuilder(args) call. Because we're using CommandLineParser and the command line, itself, consists of Verbs and their arguments, the only way to get the "everything else to pass to Host.CreateDefaultBuilder" was to use CommandLineParser's dashdash option which says "grab the rest of the args as is". This is only valid for the [StartOptions](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/CommandParserOptions/StartOptions.cs#L15).